website/docs/cloud-docs/api-docs/reserved-tag-keys.mdx

```diff
@@ -0,0 +1,273 @@
+---
+page_title: Reserved Tag Keys - API Docs - HCP Terraform
+description: >-
+    Use the `/reserved-tag-keys` API endpoints to denote tag keys that have special meaning for your organization. Reserving tag keys allows project and workspace managers can follow a consistent tagging strategy, and also provides project admins with a means of disabling overriding inherited tags.
+---
+
+[200]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200
+
+[201]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201
+
+[202]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/202
+
+[204]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/204
+
+[400]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400
+
+[401]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401
+
+[403]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/403
+
+[404]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404
+
+[409]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/409
+
+[412]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/412
+
+[422]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422
+
+[429]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/429
+
+[500]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/500
+
+[504]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/504
+
+[JSON API document]: /terraform/cloud-docs/api-docs#json-api-documents
+
+[JSON API error object]: https://jsonapi.org/format/#error-objects
+
+[speculative plans]: /terraform/cloud-docs/run/remote-operations#speculative-plans
+
+# Reserved Tag Keys API
+
+## Overview
```

Comment:
```suggestion
```

Since it's already implied and we usually don't include headers for the first introductory paragraph. it's also strange to see headers back to back

<img width="595" alt="image" src="https://github.com/user-attachments/assets/c5ea999c-f35d-424c-8a00-36659b6cf365">

Reply:
alternatively, like the projects api page we can add a single line description

```suggestion
This topic provides reference information about the projects API.

## Overview
```